hotsos/core/ssl.py

```diff
@@ -0,0 +1,46 @@
+from hotsos.core.log import log
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from datetime import datetime
+
+
+class SSLCertificate(object):
+
+    """
+    This class instantiates an SSLCertificate object.
+    """
+
+    def __init__(self, certificate_path):
+        """
+        @param certificate_path: path to the certificate that we want to load
+        in order to gather information from.
+        If the file in the path fails to read, it raises an exception.
+        """
+        self.certificate_path = certificate_path
+        try:
+            with open(self.certificate_path, "rb") as fd:
+                self.certificate = fd.read()
+        except OSError as e:
+            log.debug("Unable to read SSL certificate file %s, error: %s",
```

Comment:
use log.error if you are going to raise the exception

we might want to just raise a log.warning and not raise the exception since this will mean the whole plugin fails

Reply:
fixed on 59fc4895706a98108a6fc7c588544aceb54b1787